### ISSUE TITLE
Move to MavenCentral or JetBrains Space

### ISSUE THREAD

**Issue by hfhbd** [cy6erGn0m/kotlinx-uuid#15]
Currently, `kotlinx-uuid` is released on Bintray, which will be closed on May 01.

**Comment by hfhbd** [comment]
@cy6erGn0m Did you remove this artifact from JCenter? Suddenly, all requests are forbidden. 

Edit: Bintray is on maintenance, https://status.bintray.com.

**Comment by cy6erGn0m** [comment]
> @cy6erGn0m Did you remove this artifact from JCenter? Suddenly, all requests are forbidden. 
> 
> Edit: Bintray is on maintenance, https://status.bintray.com.

No, didn't. Actually, jcenter is going to stop working at some point so we need to think of migrating to somewhere

**Comment by ghost** [comment]
Any updates?

**Comment by hfhbd** [comment]
@andreypfau You could use my fork: https://github.com/hfhbd/kotlinx-uuid
